torch_xla/__init__.py

```diff
@@ -1,5 +1,16 @@
 import torch
+import os
```

Comment:
This has to be:
```Python
import os
os.environ['TF_GRPC_DEFAULT_OPTIONS'] = ...
...
```

*Before* importing `_XLAC`.

Reply:
Ah yes, good catch. Thanks.

Reply:
That goes before everything.
Even `import torch` ...

Reply:
Done.